dockerfiles/mariadb1011-container-disk:c10s/install_hammerdb.sh

```
@@ -1,8 +1,11 @@
-curl -LO https://github.com/TPC-Council/HammerDB/releases/download/v4.0/HammerDB-4.0-Linux.tar.gz
-tar -xf HammerDB-4.0-Linux.tar.gz
-mkdir /hammer
-mv HammerDB-4.0/* /hammer
+curl -LO https://github.com/TPC-Council/HammerDB/releases/download/v4.12/HammerDB-4.12-Linux.tar.gz
+tar -xf HammerDB-4.12-Linux.tar.gz
+mkdir -p /hammer
+mv HammerDB-4.12/* /hammer
 export LD_LIBRARY=/hammer/lib:$LD_LIBRARY_PATH
 dnf install -y tk
+rm -rf /root/HammerDB*
 cd /hammer; ./bin/tclsh8.6
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)